Documentation builds for creating PDF fail

It's currently not possible to build the PDF of the user documentation:
```
error	30-Mar-2018 15:12:36	SEVERE: Exception
error	30-Mar-2018 15:12:36	org.apache.fop.apps.FOPException: The markup declarations contained or pointed to by the document type declaration must be well-formed.
error	30-Mar-2018 15:12:36	javax.xml.transform.TransformerException: The markup declarations contained or pointed to by the document type declaration must be well-formed.
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.InputHandler.transformTo(InputHandler.java:303)
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.InputHandler.renderTo(InputHandler.java:130)
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.Main.startFOP(Main.java:177)
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.Main.main(Main.java:208)
error	30-Mar-2018 15:12:36	Caused by: javax.xml.transform.TransformerException: The markup declarations contained or pointed to by the document type declaration must be well-formed.
error	30-Mar-2018 15:12:36		at org.apache.xalan.transformer.TransformerImpl.fatalError(TransformerImpl.java:780)
error	30-Mar-2018 15:12:36		at org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:756)
error	30-Mar-2018 15:12:36		at org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:1284)
error	30-Mar-2018 15:12:36		at org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:1262)
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.InputHandler.transformTo(InputHandler.java:300)
error	30-Mar-2018 15:12:36		... 3 more
error	30-Mar-2018 15:12:36	
error	30-Mar-2018 15:12:36	---------
error	30-Mar-2018 15:12:36	
error	30-Mar-2018 15:12:36	; SystemID: http://www.docbook.org/xml/4.4/docbookx.dtd; Line#: 1; Column#: 1
error	30-Mar-2018 15:12:36	javax.xml.transform.TransformerException: The markup declarations contained or pointed to by the document type declaration must be well-formed.
error	30-Mar-2018 15:12:36		at org.apache.xalan.transformer.TransformerImpl.fatalError(TransformerImpl.java:780)
error	30-Mar-2018 15:12:36		at org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:756)
error	30-Mar-2018 15:12:36		at org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:1284)
error	30-Mar-2018 15:12:36		at org.apache.xalan.transformer.TransformerImpl.transform(TransformerImpl.java:1262)
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.InputHandler.transformTo(InputHandler.java:300)
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.InputHandler.renderTo(InputHandler.java:130)
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.Main.startFOP(Main.java:177)
error	30-Mar-2018 15:12:36		at org.apache.fop.cli.Main.main(Main.java:208)
```

This seems to be caused by docbook now requiring a https link: `https://www.docbook.org/xml/4.4/docbookx.dtd`

--- a/documentation/ascraeus_doc.xml
+++ b/documentation/ascraeus_doc.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
-                         "http://www.docbook.org/xml/4.4/docbookx.dtd" [
+                         "https://www.docbook.org/xml/4.4/docbookx.dtd" [
                          <!ENTITY % xinclude SYSTEM "includes/xinclude.mod" >
                          %xinclude;
                          <!ENTITY % local.common.attrib "xml:base CDATA #IMPLIED" >

--- a/documentation/content/en/chapters/admin_guide.xml
+++ b/documentation/content/en/chapters/admin_guide.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
-                         "http://www.docbook.org/xml/4.4/docbookx.dtd">
+                         "https://www.docbook.org/xml/4.4/docbookx.dtd">
 <chapter id="admin_guide">
 	<?dbhtml dir="adminguide" filename="index.html" ?>
 	<chapterinfo>

--- a/documentation/content/en/chapters/glossary.xml
+++ b/documentation/content/en/chapters/glossary.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
-                         "http://www.docbook.org/xml/4.4/docbookx.dtd">
+                         "https://www.docbook.org/xml/4.4/docbookx.dtd">
 <chapter id="glossary">
 	<?dbhtml dir="glossary" filename="index.html" ?>
 	<chapterinfo>

--- a/documentation/content/en/chapters/moderator_guide.xml
+++ b/documentation/content/en/chapters/moderator_guide.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
-                         "http://www.docbook.org/xml/4.4/docbookx.dtd">
+                         "https://www.docbook.org/xml/4.4/docbookx.dtd">
 <chapter id="moderator_guide">
 	<?dbhtml dir="moderatorguide" filename="index.html" ?>
 	<chapterinfo>

--- a/documentation/content/en/chapters/quick_start_guide.xml
+++ b/documentation/content/en/chapters/quick_start_guide.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
-                         "http://www.docbook.org/xml/4.4/docbookx.dtd">
+                         "https://www.docbook.org/xml/4.4/docbookx.dtd">
 <chapter id="quick_start_guide">
 	<?dbhtml  dir="quickstart" filename="index.html" ?>
 	<chapterinfo>

--- a/documentation/content/en/chapters/upgrade_guide.xml
+++ b/documentation/content/en/chapters/upgrade_guide.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
-                         "http://www.docbook.org/xml/4.4/docbookx.dtd">
+                         "https://www.docbook.org/xml/4.4/docbookx.dtd">
 <chapter id="upgrade_guide">
 	<?dbhtml dir="upgradeguide" filename="index.html" ?>
 	<chapterinfo>

--- a/documentation/content/en/chapters/user_guide.xml
+++ b/documentation/content/en/chapters/user_guide.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
-                         "http://www.docbook.org/xml/4.4/docbookx.dtd">
+                         "https://www.docbook.org/xml/4.4/docbookx.dtd">
 <chapter id="user_guide">
 	<?dbhtml dir="userguide" filename="index.html" ?>
 	<chapterinfo>

